### DISCUSSION TITLE
Do we have any plan to support Chinese GM TLS protocol?

### DISCUSSION THREAD

**Discussion by andylee521** [openssl/openssl#22978] [General Discussion]
As title.
And I'm a Chinese developer on this.
I can do a pull request based on OPENSSL-3.0 for this.

**Comment by mattcaswell** [comment]
No current plans. Is there an English language version of the spec somewhere?

I'd be interested in @InfoHunter's opinion on this.

Note: if we were to consider such a PR it would have to be against master not 3.0

**Comment by andylee521** [comment]
Hi mattcaswell,
I think it have no English version unless GMTLS protocol become to a RFC.
I'm sorry for that.

**Reply by mattcaswell** [reply]
Without a good translation it would be difficult for us to accept a PR to add this, because we could not verify the implementation against the spec - or maintain it going forwards.

**Comment by andylee521** [comment]
I got it, thank your for reply, thank you very much.
